### ISSUE TITLE
Mark all chats as read

### ISSUE THREAD

**Issue by Ahwxorg** [ulyssa/iamb#377]
Is there a way to mark all chats as read? On Element, things are already marked as read, and when I use iamb, those chats are unread.

**Comment by thibaultamartin** [comment]
You might be looking for [`:unreads clear`](https://iamb.chat/rooms/browsing.html#browsing-unreads)

**Comment by ulyssa** [comment]
Yep, `:unreads clear` does this. 😄 You'll want to make sure you're running v0.0.10 (or later, for future readers who stumble upon this issue), which contains #332.

**Comment by Ahwxorg** [comment]
This works, however it also marks chats that are actually unread as unread. I would like something that would sync the "unread" state between fluffychat/element/iamb. Does that exist in iamb?

**Comment by ulyssa** [comment]
Ah, that's a bug caused when other clients set the read receipt marker to a recent state event (such as when users join or leave, or room properties change). There's a couple other tickets open for it (#291, #292). Once iamb shows state events in the scrollback then the issue should be fixed.
